src/coprocessor/dag/expr/err.rs

```diff
@@ -138,6 +141,13 @@ impl From<Utf8Error> for Error {
     }
 }
 
+impl From<RegexpError> for Error {
+    fn from(err: RegexpError) -> Error {
+        let msg = format!("Got error '{:.64}' from regexp", err.description());
```

Comment:
Does tidb have the same error code? @XuHuaiyu

Reply:
Looks it doesn't for now, I think we'd better fix it at TiDB side

Reply:
Error code at TiDB side is fixed(https://github.com/pingcap/tidb/pull/6854)
@AndreMouche, PTAL thanks